features/other_b/series_order.feature

```diff
@@ -29,3 +29,31 @@ Feature: Rearrange works within a series
       And I should see "1. A Bad, Bad Night"
       And I should see "2. Things Get Worse"
       And I should see "3. A Bad, Bad Day"
+
+  @javascript
+  Scenario: Reordering series by drag and drop updates work blurbs correctly.
```

Comment:
Support was asking about this issue and I'm a little confused -- is the issue with the blurb (the thing that appears on work index pages) or the meta (the thing that appears on the top of individual works)? The issue and pull request say the blurb, but the test appears to be checking the meta.

Reply:
This PR fixes both issues, I have edited the test to make sure the blurb is checked.

Reply:
Thanks for clarifying! I'll update the issue to make sure everything gets tested.